admin/timescale/create_views.sql

```diff
@@ -0,0 +1,20 @@
+BEGIN;
+
+-- create a materialized view of the top mappings of a recording msid. top mappings are chosen
+-- by the highest number of users that have added it. if multiple mappings have same count, break
+-- ties by preferring the mapping that was created most recently.
+CREATE MATERIALIZED VIEW mbid_manual_mapping_top AS (
```

Comment:
The same user will see the effect because of the way the select listens query is written. Other users won't see the effect until view refresh. Yes, will add to update script.

Reply:
oh cool, I didn't notice that in the sql query. that's a neat idea